doc/building_fpga_bitstreams.md

```diff
@@ -0,0 +1,100 @@
+# Building FPGA Bitstreams for OT-SCA
+
+The idea of this tutorial is to document how to build FPGA bitstreams that can
+be used for performing side-channel analysis on the ChipWhisperer boards using
+ot-sca.
+
+## Prerequisites
+
+Bitstreams are created inside the [OpenTitan](https://github.com/lowRISC/OpenTitan)
+repository. Make sure to follow the getting started
+[guide](https://opentitan.org/book/doc/getting_started/index.html) before attempting
+to build bitstreams. A Xilinx Vivado installation with a valid license is needed.
```

Comment:
Is there a minimum version requirement for Vivado?

Reply:
I'd prefer not mentioning a specific Vivado version in this document as the linked document contains information about the version. With this approach, we do not have to keep updating the Vivado version in this document.